static/app/views/profiling/landing/profilingScatterChart.tsx

```diff
@@ -154,6 +158,48 @@ function makeScatterChartOptions({
   projects: Project[];
   theme: Theme;
 }) {
+  const user = ConfigStore.get('user');
+  const options = user?.options;
+
+  const _tooltipFormatter: TooltipComponentFormatterCallback<any> = seriesParams => {
+    const dataPoint = data[seriesParams.seriesName]?.[seriesParams.dataIndex];
+    const project = dataPoint && projects.find(proj => proj.id === dataPoint.app_id);
+
+    const entries = [
+      {label: t('Project'), value: project?.slug},
+      {label: t('App Version'), value: dataPoint?.app_version},
+      {
+        label: t('Duration'),
+        value: defined(dataPoint?.trace_duration_ms)
+          ? getDuration(dataPoint?.trace_duration_ms, 2, true)
+          : null,
+      },
+      {label: t('Interaction'), value: dataPoint?.interaction_name},
+      {label: t('Device Model'), value: dataPoint?.device_model},
+      {label: t('Device Class'), value: dataPoint?.device_class},
+      {label: t('Device Manufacturer'), value: dataPoint?.device_manufacturer},
+    ]
+      .filter(({value}) => defined(value))
```

Comment:
I would keep the undefined values - there is probably still some value in knowing what was set. E.g. if we have a bug or something on the backend, seeing undefined will help us surface it. Maybe we can just have a label fallback below in the map fn like `${label ?? "unknown"}`